addons/website_sale_product_configurator/static/src/js/website_sale_options.js

```diff
@@ -99,7 +99,7 @@ publicWidget.registry.WebsiteSale.include({
                 'item_name': el.getElementsByClassName('product_display_name')[0].textContent,
                 'quantity': parseFloat(el.getElementsByClassName('js_quantity')[0].value),
                 'currency': currency,
-                'price': parseFloat(el.getElementsByClassName('oe_price')[0].getElementsByClassName('oe_currency_value')[0].textContent),
+                'price': el.getElementsByClassName('product_price')[0].value,
```

Comment:
I think it's better to keep the `parseFloat` around this:
```suggestion
                'price': parseFloat(el.getElementsByClassName('product_price')[0].value),
```